docs/LOGGING.md

```diff
@@ -0,0 +1,49 @@
+# Logging
+
+There are two manners with which Crownstone firmware supports logging:
+
+* plain text logging
+* binary logging
+
+The firmware will send logs via UART. The UART device to be used depends on the configuration settings that you set at build or configuration time.
```

Comment:
`The UART device to be used depends on the configuration settings that you set at build or configuration time.` device?

Reply:
Yes, that is a pleonasm indeed. A UART is indeed already a device. It's a receiver-transmitter. :-) I'll reword it.

Reply:
Ah, now i see what you meant, the device on the pc side.